client/electron/scripts/sign-latest-release.py

```diff
@@ -0,0 +1,137 @@
+# Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
+
+"""
+Automate the signing of the latest release artifact.
+
+This script downloads the latest pre-built artifact from the GitHub repository,
+unzips it, checks the version, and create a signed windows installer using the
+embedded signing script.
+
+
+Requirements:
+- Python 3.9+
+- gh (GitHub CLI) installed and authenticated
+- tqdm installed (pip install tqdm)
+- urllib3>=2 installed (pip install "urllib3>=2")
```

Comment:
Also the sign script, isn't it? held by project admins? 
```suggestion
- urllib3>=2 installed (pip install "urllib3>=2")
- The sign script (sign-windows-package.cmd) held by project administrators
```

Reply:
No, the sign script is part of the artifact